### PR TITLE
Disable portable mode of VS Code

### PR DESCRIPTION
**Pull request by inosik**
See discussion in #1037.

### CODE REVIEW

File: vscode-portable.json [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        "if(Test-Path(\"$dir\\vscode-install-context.reg\")) {",
+        "  $codepath = \"$dir\\Code.exe\".Replace('\\', '\\\\')",
+        "  $content = Get-Content \"$dir\\vscode-install-context.reg\"",
+        "  $content = $content.Replace('$code', $codepath)",
+        "  $content | Set-Content -Path \"$dir\\vscode-install-context.reg\"",
+        "}"
+    ],
+    "checkver": {
+        "url": "https://code.visualstudio.com/sha?build=stable",
+        "jp": "$.products[?(@.platform.os == 'win32-x64-archive')].name"
```
Comment (rasa) [inline vscode-portable.json]: Are we sure we want to key off the `.name` attribute, and not the `.version` attribute?

Comment (r15ch13) [reply vscode-portable.json]: `.version` is just a hash. `.productVersion` and `.name` are the same.

Comment (Ash258) [reply vscode-portable.json]: `.version` is github commit, which was used to build binaries.

Comment (rasa) [reply vscode-portable.json]: Hence why it should be in the checkver. Or am I missing something?

**Pull request merged**